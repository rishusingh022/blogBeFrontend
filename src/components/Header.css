nav {
  padding-top: 2rem;
  padding-bottom: 2rem;
  background-color: #121212;
  color: white;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

nav ul {
  list-style: none;
  display: flex;
  align-items: center;
  gap: 6rem;
}

nav a {
  text-decoration: none;
  color: white;
}

@media screen and (max-width: 550px) {
  nav {
    flex-direction: column;
    gap: 1rem;
  }
}
