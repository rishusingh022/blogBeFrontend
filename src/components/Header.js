import React from "react";
import "./Header.css";
function Header() {
  return (
    <nav className="body-padding">
      <div>
        <h1>The Artifacts</h1>
        <p>
          <em>Culture & Art Blogs</em>
        </p>
      </div>
      <ul>
        <li>
          <a href="#">Blog</a>
        </li>
        <li>
            <a href="#">About</a>
        </li>
        <li>
            <a href="#">Contact</a>
        </li>
      </ul>
    </nav>
  );
}

export default Header;
