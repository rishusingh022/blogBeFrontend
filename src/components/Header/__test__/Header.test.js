import { render } from "@testing-library/react";
import Header from "../Header";
import React from "react";

describe("Header", () => {
  it("should render correctly", () => {
    const view = render(<Header />);
    expect(view.asFragment()).toMatchSnapshot();
  });
});
