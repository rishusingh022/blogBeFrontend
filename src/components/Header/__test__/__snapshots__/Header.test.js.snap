// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <nav
    class="body-padding"
  >
    <div>
      <h1>
        The Artifacts
      </h1>
      <p>
        <em>
          Culture & Art Blogs
        </em>
      </p>
    </div>
    <ul>
      <li>
        <a
          href="#"
        >
          Blog
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
